python/rmm/_lib/memory_resource.pyx

```diff
@@ -144,11 +145,36 @@ cdef class CudaMemoryResource(DeviceMemoryResource):
 cdef class CudaAsyncMemoryResource(DeviceMemoryResource):
     """
     Memory resource that uses cudaMallocAsync/Free for
-    allocation/deallocation
+    allocation/deallocation.
+
+    Parameters
+    ----------
+    initial_pool_size: int, optional
+        Initial size of the pool in bytes. If no value is provided,
+        the initial size is 0.
```

Comment:
I changed this behavior here https://github.com/rapidsai/rmm/pull/757 to make it more consistent with the other pool.

Reply:
Fixed docs to reflect this